drizzle-kit/src/cli/commands/introspect.ts

```diff
@@ -201,7 +211,7 @@ export const introspectMysql = async (
 	writeFileSync(relationsFile, relationsTs.file);
 	console.log();
 
-	const { snapshots, journal } = prepareOutFolder(out, 'postgresql');
```

Comment:
This is inside the `introspectMysql` function, so the output folder should be named `mysql` and not `postgresql`, correct?